src/index.html

```diff
@@ -11,12 +11,128 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.google.com/specimen/Roboto"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+    />
```

Comment:
The attributes of the link element should be formatted correctly. Each attribute should start on a new line with a 2-space indentation relative to the tag. Moreover, the line of code exceeds 80 characters, which can make it harder to read.